packages/ui/src/views/layouts/PullRequestCompareLayout.tsx

```diff
@@ -357,7 +356,7 @@ const PullRequestCompare: React.FC<SandboxPullRequestCompareProps> = ({
     </SandboxLayout.Main>
   )
 }
-interface HeaderProps {
+export interface HeaderProps {
```

Comment:
`HeaderProps` seems _super_ generic. I don't think we need this now in `design-system`, so we can remove the `export`, I think

Reply:
done

Reply:
done